shared/actions/users.tsx

```diff
@@ -70,6 +72,26 @@ const refreshBlockList = async (action: TeamBuildingGen.SearchResultsLoadedPaylo
     usernames: action.payload.users.map(u => u.serviceMap['keybase'] || '').filter(Boolean),
   })
 
+const submitRevokeVouch = async (_: TypedState, action: UsersGen.SubmitRevokeVouchPayload) => {
+  try {
+    await RPCTypes.revokeRevokeSigsRpcPromise(
+      {sigIDQueries: [action.payload.proofID]},
+      Constants.wotReactWaitingKey
+    )
+    return Tracker2Gen.createLoad({
+      assertion: action.payload.voucheeName,
+      forceDisplay: false,
+      fromDaemon: false,
+      guiID: TrackerConstants.generateGUIID(),
+      ignoreCache: false,
+      inTracker: false,
+      reason: '',
```

Comment:
i don't know if any of this is right

Reply:
What happens if you do pass `reason: 'wotRevokedVouch'` here?

Reply:
works just fine. will change.